cumulusci/cumulusci.yml

```diff
@@ -1,7 +1,14 @@
 cumulusci:
     keychain: cumulusci.core.keychain.EncryptedFileProjectKeychain
-
+    
 tasks:
+    activate_process_builder:
+        group: Salesforce Bulk API
+        description: "Takes a selected list of developer names and runs activating flows against these orgs"
+        class_path: cumulusci.tasks.salesforce.ActivateProcessBuilderProcesses.ActivateProcessBuilderProcesses
+        options:
+            developer-names: 
+              - ServiceDelivery
```

Comment:
I don't think we want to ship any options here - that could expose side effects in user orgs.

Reply:
Ok should I pass in as parameter?

Reply:
We just don't want the configuration to be in the global `cumulusci.yml`. We'd want to have no options declared here, and then we'd declare them in, e.g., PMDM's `cumulusci.yml` (since the options are going to be project-specific).

For testing, you can supply it on the command line, e.g., `-o developer_names ServiceDelivery`.

Reply:
ah makes sense to me